pkg/controller/migplan/validation.go

```diff
@@ -47,6 +47,8 @@ const (
 	NsLimitExceeded                            = "NamespaceLimitExceeded"
 	NsLengthExceeded                           = "NamespaceLengthExceeded"
 	NsHaveNodeSelectors                        = "NamespacesHaveNodeSelectors"
+	DuplicateNsOnSourceCluster                 = "NamespaceNotFoundOnSourceCluster"
```

Comment:
Is this a copy paste error? The content of the string doesn't seem to match the const name.

Reply:
Yes. I've already fixed it in the commit.